browsers/chrome.json

```
@@ -795,25 +795,26 @@
         "114": {
           "release_date": "2023-05-30",
           "release_notes": "https://chromereleases.googleblog.com/2023/05/stable-channel-update-for-desktop_30.html",
-          "status": "current",
+          "status": "retired",
           "engine": "Blink",
           "engine_version": "114"
         },
         "115": {
-          "release_date": "2023-07-18",
-          "status": "beta",
+          "release_date": "2023-07-21",
```

See https://chromiumdash.appspot.com/schedule 

```suggestion
          "release_date": "2023-07-18",
```